rules/windows/discovery_group_policy_object_discovery.toml

```diff
@@ -0,0 +1,43 @@
+[metadata]
+creation_date = "2023/01/18"
+integration = ["windows", "endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/01/18"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects the usage of gpresult.exe to query group policy objects. Attackers may query group policy objects during the reconnaissance phase after compromising a system to gain a better understanding of the active directory environment and possible methods to escalate privileges or move laterally. 
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*", "endgame-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Group Policy Discovery Through gpresult.exe"
+risk_score = 21
+rule_id = "94a401ba-4fa2-455c-b7ae-b6e037afc0b7"
+severity = "low"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Discovery", "Elastic Endgame"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where event.type == "start" and
+(
+(process.name: "gpresult.exe" and (process.args: "/z" or process.args: "/v" or process.args: "/r" or process.args: "/h" or process.args: "/x"))
+)
```

Comment:
```suggestion
process where event.type == "start" and
(process.name: "gpresult.exe" or process.pe.original_file_name == "gprslt.exe") and process.args: ("/z", "/v", "/r", "/x")
```

`process.args :(a, b, c) is equiv to (process.args : a or process.args : b or process.args : c)` also for process name always try to add process.pe.original_file_name for better resilience (they may rename the process). 

![image](https://user-images.githubusercontent.com/64742097/213166531-046aec29-9215-42db-95eb-1c2c75a744bd.png)

Reply:
Good calls, thanks for the feedback!